oecologia-australis.csl

```diff
@@ -142,38 +138,38 @@
         </choose>
         <text variable="number-of-pages" prefix="p. " suffix="."/>
         <group prefix=" ">
-          <text term="retrieved" text-case="capitalize-first" suffix=" "/>
-          <text term="from" suffix=" "/>
-          <text variable="URL"/>
+          <choose>
+            <if type="chapter" match="none">
+              <text term="retrieved" text-case="capitalize-first" suffix=" "/>
+              <text term="from" suffix=" "/>
+              <text variable="URL"/>
+            </if>
+          </choose>
         </group>
       </if>
+      <else-if type="legislation webpage" match="any">
+        <text value="Retrieved in" prefix="("/>
+        <date form="text" variable="accessed" prefix=" " suffix=", ">
+          <date-part name="day" form="ordinal"/>
+          <date-part name="month"/>
+          <date-part name="year"/>
+        </date>
+        <text value="from " prefix=" "/>
+        <text variable="URL" strip-periods="false" quotes="false" font-variant="normal" suffix=")."/>
```

Comment:
I think not, because only legislation or webpages are used between parenthesis. If I put it as group role, all it will be included between parenthesis. Furthermore, if someone forget these information, it would be easier to found an error. There is some better method to solve this?